docs/meshstack.aws.aad-as-idp.md

```diff
@@ -0,0 +1,80 @@
+---
+id: meshstack.aws.aad-as-idp
+title: AAD as Identity Provider
```

Comment:
d: I think we should call this page "AWS SSO Setup". And then inside the page you could initially mention, that currently only the AAD setup is described here.

Reply:
Changed to AWS SSO Setup